swagger-apis/accounts/2.0.0-RC1.0.yml

```
@@ -396,20 +396,20 @@ components:
         compeCode:
           type: string
           maxLength: 3
-          pattern: '\d{3}|^NA$'
+          pattern: '\d{3}$'
```

@Lucas-Silva75 Acredito que podemos alterar o pattern para ^\d{3}$, o pattern que existe está aceitando uma quantidade de digitos além do permitido no campo. O mesmo vale nos outros campos
```suggestion
          pattern: '^\d{3}$'
```

---

Bem observado @cleiltonoliveira , irei ajustar aqui!